.circleci/config.yml

```diff
@@ -7,8 +7,172 @@ orbs:
+              exit 0
+          else
+              exit $http_code
+          fi
+          docker kill taraxa-node-smoke-test-${DOCKER_BRANCH_TAG} || true
+          docker network rm smoke-test-net-${DOCKER_BRANCH_TAG} || true
+          docker images -a
+      - run:
+         name: Add Tags and push images
+         command: |
+           echo ${GCLOUD_SERVICE_KEY} | docker login -u _json_key --password-stdin https://gcr.io
+           docker tag ${IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} ${GCP_IMAGE}:${CIRCLE_SHA1}
+           docker tag ${IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} ${GCP_IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM}
+           docker tag ${IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} ${GCP_IMAGE}:${START_TIME}
+           docker tag ${IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} ${GCP_IMAGE}:${START_TIME}-${CIRCLE_SHA1}
+           docker push ${GCP_IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM}
+           docker push ${GCP_IMAGE}:${START_TIME}
+           docker push ${GCP_IMAGE}:${CIRCLE_SHA1}
+           docker push ${GCP_IMAGE}:${START_TIME}-${CIRCLE_SHA1}
+           if [[ ! ${CIRCLE_BRANCH} =~ ^PR-* ]];then
+              docker tag ${IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} ${GCP_IMAGE}:latest
+              #docker push ${GCP_IMAGE}:latest
+           fi
+           if [[ ${CIRCLE_BRANCH} == "master" ]];then
+             echo ${DOCKERHUB_PASS} | docker login -u taraxa --password-stdin
+             docker push ${IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM}
+             docker push ${IMAGE}:${START_TIME}
+             docker push ${IMAGE}:${CIRCLE_SHA1}
+             docker push ${IMAGE}:${START_TIME}-${CIRCLE_SHA1}
+             if [[ ! ${CIRCLE_BRANCH} =~ ^PR-* ]];then
+               docker tag ${IMAGE}:${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} ${IMAGE}:latest
+               #docker push ${IMAGE}:latest
+             fi
+           fi
+      - run:
+         name: Install/configure helm and chart
+         command: |
+            if [[ ${CIRCLE_BRANCH} =~ ^PR-* ]];then
+              git clone --branch development  https://github.com/Taraxa-project/taraxa-testnet.git
+              mkdir -p $HOME/.kube
+              echo -n "${KUBE_CONFIG}" | base64 --decode > $HOME/.kube/config
+              curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+              helm install --name ${HELM_TEST_NAME} taraxa-node \
+                --wait \
+                --atomic \
+                --timeout 1200 \
+                --namespace ${HELM_TEST_NAME} \
+                --set replicaCount=5 \
+                --set test.pythontester.script=jenkins.py \
+                --set image.repository=${GCP_IMAGE} \
+                --set image.tag=${DOCKER_BRANCH_TAG}-${CIRCLE_BUILD_NUM} \
+                -f taraxa-node/values.yaml
+            fi
+      - run:
+         name: Run helm test
+         command: |
+            if [[ ${CIRCLE_BRANCH} =~ ^PR-* ]];then
```

Comment:
As discussed, I think circle ci don't show PRs as branches like this
So we need to fix this